src/interfaces.rs

```diff
@@ -13,26 +13,58 @@ use editor::{Rect, KeyCode, Key, KnobMode};
 use host::Host;
 
 /// Deprecated process function.
-pub fn process_deprecated(_effect: *mut AEffect, _raw_inputs: *const *const f32, _raw_outputs: *mut *mut f32, _samples: i32) { }
+pub fn process_deprecated(
+    _effect: *mut AEffect,
+    _raw_inputs: *const *const f32,
+    _raw_outputs: *mut *mut f32,
+    _samples: i32,
+) {
+}
 
 /// VST2.4 replacing function.
-pub fn process_replacing(effect: *mut AEffect, raw_inputs: *const *const f32, raw_outputs: *mut *mut f32, samples: i32) {
+pub fn process_replacing(
+    effect: *mut AEffect,
+    raw_inputs: *const *const f32,
+    raw_outputs: *mut *mut f32,
+    samples: i32,
+) {
     // Handle to the vst
```

Comment:
VST in documentation is capitalized, so might as well keep it that way to stay consistent.